app/views/topics/index.rss.builder

```diff
@@ -2,7 +2,7 @@
 xml.instruct! :xml, :version => "1.0"
 xml.rss :version => "2.0" do
   xml.channel do
-    xml.title "#{Settings.site_name} #{@forum.name} Feed"
+    xml.title "#{AppSettings['settings.site_name']} #{@forum.name} Feed"
```

Comment:
Unfortunately this is the API that is available from the rails settings cached gem.  It is not the prettiest way to express a namespaced variable.  About the only ooption would be to use a different name.